tests/magnetostatics/test_biot_savart.py

```diff
@@ -56,6 +56,7 @@ def test_biot_savart_loop():
     circle = make_circle(radius=x_coil, center=(0, z_coil, 0), axis=(0, 0, 1))
     filament = circle.discretize(ndiscr=2000)
     bsf = BiotSavartFilament(filament, radius)
+    bsf.plot()
```

Comment:
Although this is closed in the `plot_errors` function we should really close this one here or at the end of this function. I should really get around to #1320...
```suggestion
    bsf.plot()
    plt.show()
    plt.close()
```